### PR TITLE
Address Sam's review comments

### PR DESCRIPTION
**Pull request by olivdb**

### CODE REVIEW

File: contracts/l2/L2DAIWormholeBridge.sol [hunk 1]
```diff
@@ -85,40 +80,40 @@ contract L2DAIWormholeBridge is OVM_CrossDomainEnabled {
...
     );
-    sendCrossDomainMessage(l1DAITokenBridge, l1Gas, message);
+    sendCrossDomainMessage(l1DAIWormholeBridge, 0, message);
 
     emit WormholeInitialized(wormhole);
   }
 
   function flush(bytes32 targetDomain) external {
     uint256 daiToFlush = batchedDaiToFlush[targetDomain];
+    require(daiToFlush > 0, "L2DAIWormholeBridge/no-dai-to-flush");
```
Comment (hexonaut) [inline contracts/l2/L2DAIWormholeBridge.sol]: Thinking more on this, I think we should allow 0 DAI flushes. I'm thinking likely we will need to keep track of the debt, settlements on the target domain over time to do bad debt settlement. Using the flush / settle functions as the trackers seems likely to me. Having a daily heartbeat regardless of if there is DAI to push is probably a good idea.

**Review by hexonaut — CHANGES_REQUESTED** [review]
Overall I think this covers everything. I've changed my mind about the DAI needing to be flushed being non-zero.

File: contracts/l2/L2DAIWormholeBridge.sol (hunk 1 above)
Comment (olivdb) [reply contracts/l2/L2DAIWormholeBridge.sol]: Yes, that makes sense

**Review by hexonaut — APPROVED** [review]
LGTM

**Pull request merged**